docs/examples/binding_lifetime/binding_lifetime.rst

```diff
@@ -0,0 +1,233 @@
+    
+    plt.figure()
+    kymo2.plot("g", aspect = 5, adjustment=lk.ColorAdjustment([0], [5]))
+
+.. image:: kymo2.png
+
+Track the binding events
+------------------------
+
+Select the region that you would like to track and load the selection in the kymotracker, :class:`~lumicks.pylake.KymoWidgetGreedy`. In this example, we crop the beads from the kymograph using `crop_by_distance`.::
+
+    kymo1_selection = kymo1.crop_by_distance(4.9,13.8)
+    kymotracker1 = lk.KymoWidgetGreedy(kymo1_selection, "green", axis_aspect_ratio=2, pixel_threshold=6, min_length=4, track_width=0.4, vmax=10)
+
+.. image:: kymotracker.png
+
+The kymotracker allows for manual adjustements. Click `Track All` and inspect the tracks. Perform manual adjustments if needed.
+
+Track the binding events on the second kymograph::
+
+    kymo2_selection = kymo2.crop_by_distance(4.9,13.8)
+    kymotracker2 = lk.KymoWidgetGreedy(kymo2_selection, "green", axis_aspect_ratio=2, pixel_threshold=6, min_length=4, track_width=0.4, vmax=10)
+
+If the the kymotracking parameters have been optimized and if manual adjustements are not needed it is also possible to track using :func:`~lumicks.pylake.track_greedy`.
+
+Save and load tracks
+--------------------
+
+The coordinates and intensities of the tracks can be saved csv, for example as `kymotracker1.tracks.save("tracks1.csv", sampling_width=3, correct_origin=True)`.`
+
+For this example, the tracking was already performed and included in the downloaded data set.
+To load tracks into Pylake use::
+
+    tracks1 = lk.load_tracks("tracks1.csv",  kymo1.crop_by_distance(4.9,13.8), "green")
+    tracks2 = lk.load_tracks("tracks2.csv",  kymo1.crop_by_distance(4.9,13.8), "green")
+
+Tracking in Lakeview
+^^^^^^^^^^^^^^^^^^^^
+Tracking can also be performed in Lakeview, exported and then loaded into Pylake for further analysis. 
+Use the same approach as above to load the tracks from Lakeview, except that the part `.crop_by_distance()` has to be removed.
+
+Combine tracks
+--------------
+
+Tracks from multiple kymographs can be combined by adding them together. Note that imagining settings such as the line time and pixel time should be the same as further explained in :ref:`global_analysis``::
+
+    tracks_total = tracks1 + tracks2
+
+The total number of tracks is::
+
+    >>> print(len(tracks_total))
+    134
+
+Fit an exponential distribution
+-------------------------------
+
+Fit an expontial distribution and compare it to the performance of a double exponential fit.
+
+.. math::
+
+    P(t) = ke^{-kt} = \frac{1}{\tau} e^{-t/\tau}
+
+Fit a single exponential to the dwell times by setting `n_components = 1` and plot the result::
+
+    exponential_fit = tracks_total.fit_binding_times(n_components = 1, observed_minimum = False, discrete_model = True)
+
+    plt.figure()
+    exponential_fit.hist()
+
+.. image:: exponential_fit.png
+
+The parameters `observed_minimum` and `discrete_model` are further explained in :ref:`dwelltime_analysis`.
```

Comment:
I feel that adding this sentence here does not make it more clear than going through the section that properly explains. The term 'discrete model' is not common and does not add much as an explanation, without more context.

Reply:
Yeah, that's fair.

I feel the section that you are referring them to is quite technical, and won't help especially beginners much in its current state. To be honest, I would almost argue that the section that is there now should probably move to `Theory` and the tutorial section should probably have something more digestible there.

Anyways, maybe I didn't phrase it very well, but I think it would be nice to tell them when you set discrete to `True`.